Src/CSharpier/CommandLineFormatter.cs

```diff
@@ -22,6 +22,7 @@ public class CommandLineFormatter
         protected readonly IFileSystem FileSystem;
         protected readonly IConsole Console;
         protected readonly IgnoreFile IgnoreFile;
+        protected readonly ILogger Logger;
```

Comment:
You should take `ILogger<CommandLineFormatter>`.
That way, you can set an option for it to print the scope/source of the message should you chose to later.

Reply:
Unless an IOC container gets passed around I don't see how that would be possible